tools/opt/opt.cpp

```diff
@@ -112,28 +126,119 @@ NOTE: The optimizer is a work in progress.
                Join two blocks into a single block if the second has the
                first as its only predecessor. Performed only on entry point
                call tree functions.
-  -h, --help   
+  -O
+               Optimize for performance. Apply a sequence of transformations
+               in an attempt to improve the performance of the generated
+               code. This flag is equivalent to specifying the following
+               flags: %s
+  -Os
+               Optimize for size. Apply a sequence of transformations in an
+               attempt to minimize the size of the generated code. This flag
+               is equivalent to specifying the following flags: %s
+  -Oconfig=<file>
+               Apply the sequence of transformations indicated in <file>.
+               This file contains a sequence of strings separated by whitespace
+               (tabs, newlines or blanks). Each string is one of the flags
+               accepted by spirv-opt. Optimizations will be applied in the
+               sequence they appear in the file. This is equivalent to
+               specifying all the flags on the command line. For example,
+               given the file opts.cfg with the content:
+
+                --inline-entry-points-exhaustive
+                --eliminate-dead-code-aggressive
+
+               The following two invocations to spirv-opt are equivalent:
+
+               $ spirv-opt -Oconfig=opts.cfg program.spv
+
+               $ spirv-opt --inline-entry-points-exhaustive \
+                    --eliminate-dead-code-aggressive program.spv
+
+               Note that the flags -O, -Os and -Oconfig=<file> are also
+               affected by other flags present on the command line. For
```

Comment:
Can we improve the "affected" phrasing?  How about:

Is this better?

> The -O, -Os, and -Oconfig flags act as macros.  Using one of them is equivalent to explicitly inserting the underlying flags at that position in the command line.

Reply:
Done.